vcam-util.c

```diff
@@ -20,14 +20,24 @@ const struct option long_options[] = {
     {NULL, 0, NULL, 0}};
 
 const char *help =
-    " -h --help                    Print this informations.\n"
-    " -c --create                  Create a new device.\n"
-    " -m --modify  idx             Modify a device.\n"
-    " -r --remove  idx             Remove a device.\n"
-    " -l --list                    List devices.\n"
-    " -s --size    WIDTHxHEIGHT    Specify resolution.\n"
-    " -p --pixfmt  pix_fmt         Specify pixel format (rgb24,yuyv).\n"
-    " -d --device  /dev/*          Control device node.\n";
+    " -h --help                            Print this informations.\n"
+    " -c --create                          Create a new device.\n"
+    " -m --modify  idx                     Modify a device.\n"
+    " -r --remove  idx                     Remove a device.\n"
+    " -l --list                            List devices.\n"
+    " -s --size    WIDTHxHEIGHTxCROPRATIO  Specify virtual resolution or crop "
```

Comment:
Would it be better to use "dimension" rather than "size"?

Reply:
I think size were still appropriate for me because the crop ratio can also represent the change of size.

And also, `-d` is already register by `--device`. 
(The reason I merge crop ratio into `-s` is the same result that `-c` and `-r` has already be register too.)